### PR TITLE
vmui: fix auto-completion triggers 

### PR DESCRIPTION
**Pull request by Loori-R**
### Describe Your Changes

- Fixes auto-complete triggers according to [these comments](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5866#issuecomment-2065273421).
- Fixes loading and displaying suggestions when there is no metric in the expression. 
   Related issue: #6153
- Adds quotes when inserting label values. 
   Related issue: #6260

### Checklist

The following checks are **mandatory**:

- [x] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Comment by hagen1778** [comment]
Hey @Loori-R! When adding label filters in curly braces, could they be filtered by metric name? In my case, autocomplete suggests labels not related to the metric name:
image

**Comment by Loori-R** [comment]
@hagen1778, fixed: now when adding label filters in curly braces, they are filtered by metric name

File: app/vmui/packages/vmui/src/components/Configurators/QueryEditor/QueryEditorAutocomplete.tsx [hunk 1]
```diff
@@ -176,6 +192,7 @@ const QueryEditorAutocomplete: FC<QueryEditorAutocompleteProps> = ({
 
   return (
     <>
+      {metric}
```
Comment (hagen1778) [inline app/vmui/packages/vmui/src/components/Configurators/QueryEditor/QueryEditorAutocomplete.tsx:195]: Is this needed here? It looks weird in UI
image

**Pull request merged**